tikv-client/src/main/java/com/pingcap/tikv/meta/TiIndexInfo.java

```diff
@@ -40,9 +41,10 @@
   private final SchemaState schemaState;
   private final String comment;
   private final IndexType indexType;
-  private final long indexColumnLength;
   private final boolean isFakePrimaryKey;
 
+  private long indexColumnSize = 8;
```

Comment:
A comment is nice to have.

Reply:
perhaps i can delete this initialization....

Reply:
done.